### PR TITLE
Update documentation for NCBIWWW.py

### PR DESCRIPTION
**Pull request by christiam**
- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

This PR updates the documentation for NCBIWWW.py , which no longer supports cloud service providers (per https://blast.ncbi.nlm.nih.gov/doc/blast-help/urlapi.html#urlapi).

### CODE REVIEW

**Comment by peterjc** [comment]
Indeed, https://github.com/ncbi/blast-cloud was archived on Aug 14, 2024.

I wonder if https://github.com/ncbi/elastic-blast aka https://blast.ncbi.nlm.nih.gov/doc/elastic-blast/ has a similar URL based API?

File: Bio/Blast/NCBIWWW.py [hunk 1]
```diff
@@ -122,7 +116,7 @@ def qblast(
      - matrix_name    Specify an alt. matrix (PAM30, PAM70, BLOSUM80, BLOSUM45).
      - filter         "none" turns off filtering.  Default no filtering
      - format_type    "HTML", "Text", "ASN.1", or "XML".  Def. "XML".
-     - entrez_query   Entrez query to limit Blast search
+     - entrez_query   Entrez query to limit Blast search - NO LONGER SUPPORTED
```
[the remark this answers is not in the document]
Comment (christiam) [reply Bio/Blast/NCBIWWW.py]: That comment was incorrect and has been amended (it's still supported for nucleotide BLAST databases).

**Comment by christiam** [comment]
> I wonder if https://github.com/ncbi/elastic-blast aka https://blast.ncbi.nlm.nih.gov/doc/elastic-blast/ has a similar URL based API?

No, it does not.

**Review by peterjc — APPROVED** [review]

**Comment by peterjc** [comment]
Thank you!

**Pull request merged**